cubids/cli.py

```diff
@@ -17,14 +17,40 @@
 
 
 def _path_exists(path, parser):
-    """Ensure a given path exists."""
+    """Ensure a given path exists.
+
+    Parameters
+    ----------
+    path : str
+        Path to check for existence.
+    parser : argparse.ArgumentParser
+        Argument parser object.
+
+    Returns
+    -------
+    pathlib.Path
+        Absolute path to the given location.
+    """
     if path is None or not Path(path).exists():
         raise parser.error(f"Path does not exist: <{path}>.")
     return Path(path).absolute()
 
 
 def _is_file(path, parser):
-    """Ensure a given path exists and it is a file."""
+    """Ensure a given path exists and it is a file.
+
+    Parameters
+    ----------
+    path : str
+        Path to check for existence.
+    parser : argparse.ArgumentParser
+        Argument parser object.
+
+    Returns
+    -------
+    pathlib.Path
+        Absolute path to the given location.
+    """
     path = _path_exists(path, parser)
     if not path.is_file():
```

Comment:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_file this suggests that all should be fine as long as it's a file or a resolvable symlink.